ospo-book/chapters/03-chapter.md

```diff
@@ -1 +1,146 @@
+
+This is a simplified view of the layers of work in an OSPO. However, depending on the complexity of your organization and the resources available to your OSPO, these layers can become more granular and include additional layers with different names.
+
+### The Pillars of Support
+
+![ospo-support](https://github.com/todogroup/ospology/assets/43671777/f96cd4a1-0315-4a0e-8de3-2da59378a57b)
+
+* Open Source Enabler: OSPOs can help organizations navigate the cultural, process, and tool changes required to engage with the open source community effectively. This can involve educating teams/ units, establishing new processes and workflows, and adopting new tools and technologies.
+
+* Open Source Counselor: OSPOs can provide guidance and advice on the latest open source trends, licensing issues, and how to engage with open source projects, foundations, and communities. This can help organizations stay up-to-date with the rapidly changing open source landscape and ensure they are making informed decisions.
+
+* Open Source Advocate: OSPOs can promote the use and/or contribution of open source and best practices across different organizational units. This can help organizations realize the benefits of open source as well as engaging people to contribute to open source projects or start new ones.
+
+* Open Source Environmentalist: OSPOs can help organizations support and sustain open source projects in the long term by addressing issues such as security, maintenance, and project health. This can involve establishing policies and procedures for code review, security vulnerability management, and ongoing maintenance and support through funding and/or contributions. By doing so, OSPOs can help ensure that open source projects remain healthy and continue to benefit the wider community.
+
+
+## Creating and Implementing an Open Source Strategy
+
+The way the people behind an OSPO achieve this is by creating and maintaining a framework covering the following aspects: strategy, governance, compliance, and community engagement. 
+The OSPO's strategy focuses on aligning the organization's open source goals with its overall organization objectives.
+
+A strategy creates a high-level consensus on concrete topics and their impact on your organization and the people within it. A good practice is to [document this strategy in an open source strategy document](https://todogroup.org/guides/strategy/).
+
+It is recommended that this document includes a general Q&A section. Additionally, you may consider creating specific how-to guides for specialized areas that interact with the OSPO and open source projects (e.g., marketing, legal, engineering) to address specific challenges. These guides should be tailored to the domain knowledge of the team members involved.
+
+### Making Open Source Relevant in the organization
```

Comment:
I don't think that making open source relevant is a task or objective of an OSPO. Open source is a means to an end for almost all organizations, so it's about facing the realities of open source usage and making the best of it to serve the business or purpose of the organization. So maybe it could be "activities directed inwards" and "activities directed outwards" or something like that.

Looking at it from a perspective of looking to the inside of the organization on one hand, and to the outside of the organization on the other hand, makes a lot of sense, though.

Reply:
I agree with the comment maybe "Understanding the relevance of open source in the organisation"?